library/src/main/camera2/com/google/android/cameraview/Camera2.java

```diff
@@ -647,13 +652,15 @@ private void startCaptureSession() {
             @SuppressWarnings("ConstantConditions")
             android.util.Size[] supportedOutputSizes = map.getOutputSizes(MediaRecorder.class);
             mVideoSize = chooseVideoSize(supportedOutputSizes, mMinVideoWidth, mMinVideoHeight);
-            mPreviewRequestBuilder = mCamera.createCaptureRequest(CameraDevice.TEMPLATE_PREVIEW);
-            mPreviewRequestBuilder.addTarget(surface);
             if (mStartVideoRecording) {
+                mPreviewRequestBuilder = mCamera.createCaptureRequest(CameraDevice.TEMPLATE_RECORD);
```

Comment:
Hmm, this is good! I wonder if this was the reason why some of the video recorded had a very long first frame.